debian/gbp.conf

```diff
@@ -1,2 +1,6 @@
 [DEFAULT]
 debian-branch = ubuntu-dock
+upstream-branch = master
```

Comment:
Also note that as we have inline changes that will be kept inline, it's another argument to not have that line + native versionning.

Reply:
Well, mostyl because I didn't like to have `gbp build-package` to be wrong (as per `--help`):

```
--git-upstream-branch=UPSTREAM_BRANCH
                        Upstream branch, default is 'upstream
```

Reply:
Well, in some cases, this is used to compute a diff between your branch and this one. Shuoldn't be an issue for a native package, but still wrong as well. Upstream is the current branch, it's a native package.

Please, really, drop it.

Reply:
yeah, that's already the case locally. Just mentioned the rationale for my initial change (especially when I was trying with quilt source format).